.github/actions/xahau-ga-dependencies/action.yml

```diff
@@ -42,6 +42,26 @@ runs:
         SAFE_BRANCH=$(echo "${{ github.ref_name }}" | tr -c 'a-zA-Z0-9_.-' '-')
         echo "name=${SAFE_BRANCH}" >> $GITHUB_OUTPUT
 
+    - name: Check conanfile changes
+      if: inputs.cache_enabled == 'true'
+      id: check-conanfile-changes
+      shell: bash
+      run: |
+        # Check if we're on the main branch
+        if [ "${{ github.ref_name }}" == "${{ inputs.main_branch }}" ]; then
+          echo "conanfile-changed=true" >> $GITHUB_OUTPUT
```

Comment:
pedantry: maybe save it as should-save-conan-cache or similar, as the name doesn't actually reflect whether the conanfile-changed